Makefile

```diff
@@ -747,6 +750,11 @@ npm-cache:
 	touch -c node_modules
 
 node_modules:
+ifneq ($(NODEJS),)
+ifneq ($(NODE_VERSION),$(shell $(NODEJS) --version | sed 's/\(v[0-9]*\.[0-9]*\)\.[0-9]*/\1/'))
```

Comment:
oh ... I see , $$ and FS, that's why I used sed instead of awk.  I would prefer awk, too!

I'm not sure all of our current node modules 100% support latest node version, so I would still use v0.10 as try server use.